README.md

```diff
@@ -1,32 +1,44 @@
-***DISCLAIMER: This codebase is not yet audited***
```

Comment:
1. Variable names in tests have been changed despite the PR description saying they are not intended to be updated. 
2. There are reference to "GovernanceStaker" in natspecs of `IEarningPowerCalculator.sol` and `INotifiableRewardReceiver.sol`

Reply:
> Variable names in tests have been changed despite the PR description saying they are not intended to be updated.

Hmm which ones? Not seeing anything obvious after a quick glance through the diff.

> There are reference to "GovernanceStaker" in natspecs of IEarningPowerCalculator.sol and INotifiableRewardReceiver.sol

Good catch! Fixded.